### PR TITLE
Fixed path dependencies to config files

### PR DESCRIPTION
**Pull request by hello-vinitha**
## Description

`stretch_ai` install path is `/home/hello-robot/.local/lib/python3.10/site-packages/stretch` and does not contain the config files. This causes `goto_controller` to crash. The PR moves the `config` folder into `src/stretch_ai` and updates the `utils/config.py` accordingly.

## Testing
- \[ \] Pull this branch and reinstall `stretch_ai`
- \[ \] Verify `/home/hello-robot/.local/lib/python3.10/site-packages/stretch/config` exists and contains the appropriate `.yaml` files.
- \[ \] Verify that `goto_controller` launches successfully: `ros2 run stretch_ros2_bridge goto_controller`

## Checklist

- \[ \] I have performed a self-review of my code
- \[ \] If it is a core feature, I have added thorough tests
- \[ \] I have added documentation for the changes
- \[ \] I have updated the README file if necessary
- \[ \] I have run on hardware if necessary

## Screenshots (if applicable)

Add any relevant screenshots or screen recordings to help reviewers understand the changes.

## Additional context

Add any other context or information about the pull request here.

### CODE REVIEW

**Comment by hello-vinitha** [comment]
Tested on `3030`

File: src/config/urdf/planner_calibrated.urdf
Comment (hello-cpaxton) [inline src/config/urdf/planner_calibrated.urdf:1]: Why was this file deleted?

File: src/config/urdf/stretch_manip_mode.urdf
Comment (hello-cpaxton) [inline src/config/urdf/stretch_manip_mode.urdf:1]: Why was this file deleted?

**Review by hello-cpaxton — APPROVED** [review]
LGTM
